CUDADataFormats/Track/test/BuildFile.xml

```diff
@@ -1,19 +1,7 @@
 <use name="HeterogeneousCore/CUDAUtilities"/>
 
-<bin file="TrackSoAHeterogeneous_t.cpp">
-  <use name="eigen"/>
-  <flags CXXFLAGS="-g -DGPU_DEBUG"/>
-</bin>
-
-<bin file="TrajectoryStateSOA_t.cpp" name="cpuTrajectoryStateSOA_t">
-  <use name="eigen"/>
-  <flags CXXFLAGS="-g -DGPU_DEBUG"/>
-</bin>
-
+<use name="CUDADataFormats/Track"/>
+<flags CUDA_FLAGS="-g -DGPU_DEBUG"/>
 <iftool name="cuda-gcc-support">
-<bin file="TrajectoryStateSOA_t.cu" name="gpuTrajectoryStateSOA_t">
-  <use name="eigen"/>
```

Comment:
What about the `TrajectoryStateSOA_t` tests ?

Reply:
Re-added, after being adapted